src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -49,6 +49,7 @@ enums:
         meaning: rdfs:Datatype
       rdf property:
         meaning: rdf:Property
+      sssom literal: A special type of literal that is used in SSSOM files to express that the subject_id is not an entity reference.
```

Comment:
current path based on your proposal @gouttegd:

1. We only allow literals in the subject position.
2. We introduce a `sssom literal` as a special case in the already existing `entity_type` field (as you suggested)
3. If that type is set, "literal mapping" mode is active (with a small number of new fields, see below)

Reply:
> We only allow literals in the subject position.

Why? Seems like an needless complication to me.

In particular, this means that literal mappings cannot possibly be inverted. And I don’t know about the use cases of those who would manipulate literal mappings, but personally I invert (non-literal) mappings _all the time_.

Reply:
OK, saw your comment below about how this would “go against the whole idea of the format”.

Not convinced, sorry. Once we admit that we can have mapping where one side is a literal rather than an entity, I don’t see how it matters which side is the literal.

Reply:
ooooookeeeeeeee :P I will fix it. Now sleeping time! :D